pkg/controller/clusterpool/clusterpool_controller.go

```diff
@@ -234,6 +278,37 @@ func (r *ReconcileClusterPool) Reconcile(request reconcile.Request) (reconcile.R
 	return reconcile.Result{}, nil
 }
 
+func (r *ReconcileClusterPool) hibernateReadyClusters(cds []*hivev1.ClusterDeployment) ([]*hivev1.ClusterDeployment, error) {
```

Comment:
This function returns ClusterDeployments for which syncsets have not been applied. Should we name it to suggest that?